app/screens/channel_info/channel_info_header.js

```diff
@@ -30,9 +33,39 @@ export default class ChannelInfoHeader extends React.PureComponent {
         type: PropTypes.string.isRequired,
         isArchived: PropTypes.bool.isRequired,
         isBot: PropTypes.bool.isRequired,
+        hasGuests: PropTypes.bool.isRequired,
         isGroupConstrained: PropTypes.bool,
     };
 
+    renderHasGuestText = (style, hasGuests, type) => {
```

Comment:
my above comment can be changed to something like this... maybe create a component for this so it is re-usable?